src/sass/Detail.scss

```diff
@@ -1,45 +1,66 @@
 .detail {
   width: 100%;
   min-width: 300px;
-  padding: 112px 0;
+  padding: 112px 0 0 0;
   display: flex;
   flex-direction: row;
   justify-content: space-between;
   .image {
-    padding-right: 137px;
+    margin-right: 100px;
+    height: 420px;
+    background-size: cover;
```

Comment:
It's convenient way to keep images with different dimensions in the same frame (I assume that pictures from users have different sizes)